CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+#### 1.38.0 - 2022/11/28
+
+### Changed
+
+* Added IPartitionKeyStrategy interface and a basic implementation, RandomPartitionKeyStrategy.
+* Included IPartitionKeyStrategy into SimpleTaskProcessingPolicy
```

Comment:
Please add one quick line why you did this. What is the benefit? Look at the other lines, they have smth like

> Reworked paranoid tasks cleaner **to work with latest mariadb drivers.**

Reply:
Done